unit-tests/live/safety_camera/test-set-get-safety-zones.py

```diff
@@ -0,0 +1,83 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device S585*
+
+import pyrealsense2 as rs
+from rspy import devices, log, test, file, repo
+
+
+#############################################################################################
+# Help Functions
+#############################################################################################
+
+
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Init")
+try:
+    ctx = rs.context()
+    dev = ctx.query_devices()[0]

```

Comment:
Why is this that needed?
You can do it outside a test, if it failes, libCI will fail IMO
also please use our testing infra helping function instead:
https://github.com/IntelRealSense/librealsense/blob/923253f645e867c75e3c6c30a84127f41d82f3e9/unit-tests/live/live-common.h#L51